File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Số lớn nhất, nhỏ nhất trong 3 số/MAX3SO.cpp
#include<bits/stdc++.h>
using namespace std;
int main() 
{
    int a, b, c;
    cin >> a >> b >> c;
    int max = a;
    if(max < b)
        max = b;
    if(max < c)
        max = c;
    int min = a; 
    if(min > b)
        min = b;
    if(min > c)
        min = c;
    cout << max << " " << min << endl;
    return 0;
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Số lớn nhất, nhỏ nhất trong 3 số/MAX3SO.cs
using System;
namespace Max3so
{
    public class Program
    {
        static void Main(string[]args)
        {
            string [] input = Console.ReadLine().Split(' ');
            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]);
            int c = int.Parse(input[2]);
            int Max = a;
            if(Max < b)
                Max = b;
            if(Max < c)
                Max = c;
            int Min = a;
            if(Min > b)
                Min = b;
            if(Min > c)
                Min = c;
            Console.Write("{0} {1}", Max, Min);
        }
    }
}